markdown_it/tree.py

```diff
@@ -272,13 +272,13 @@ def tag(self) -> str:
         return self._attribute_token().tag
 
     @property
-    def attrs(self) -> Dict[str, Any]:
+    def attrs(self) -> Dict[str, Union[str, int, float]]:
         """Html attributes."""
-        token_attrs = self._attribute_token().attrs
-        if token_attrs is None:
-            return {}
-        # Type ignore because `Token`s attribute types are not perfect
-        return dict(token_attrs)  # type: ignore
+        return self._attribute_token().attrs
+
+    def attrGet(self, name: str) -> Any:
```

Comment:
My preference would be to not add this method, but use `SyntaxTreeNode.attrs.get()` instead. Much more pythonic, and since this class is new API, there is nothing forcing us to copy the `Token` methods (which only exist because `attrs` was not a mapping in JS upstream).